.github/workflows/native-unix.yml

```diff
@@ -392,7 +392,7 @@ jobs:
         with:
           fetch-depth: 0
           persist-credentials: false
-      - uses: 'google-github-actions/auth@v2'
+      - uses: google-github-actions/auth@6fc4af4b145ae7821d527454aa9bd537d1f2dc5f  # v2
```

Comment:
```suggestion
      - uses: google-github-actions/auth@71f986410dfbc7added4569d411d040a91dc6935  # v2.1.8
```

Reply:
Hmm, I wonder how the hook missed that...